classloader-leak-prevention/classloader-leak-prevention-core/src/main/java/se/jiderhamn/classloader/leak/prevention/cleanup/ReactorNettyHttpResourcesCleanUp.java

```diff
@@ -13,7 +13,7 @@
 public class ReactorNettyHttpResourcesCleanUp implements ClassLoaderPreMortemCleanUp {
   @Override
   public void cleanUp(ClassLoaderLeakPreventor preventor) {
-    final Class<?> clazz = preventor.findClass("reactor.netty.http.HttpResources");
+    final Class<?> clazz = preventor.findClass("reactor.ipc.netty.http.HttpResources");
```

Comment:
I'm under the impression the class was renamed `reactor.netty.http.HttpResources` in version 0.8.0-SNAPSHOT. Unless the leak is also fixed in that version, I suppose we should handle both.

Reply:
You're right: [HttpResources 0.7.8 tag](https://github.com/reactor/reactor-netty/blob/v0.7.8.RELEASE/src/main/java/reactor/ipc/netty/http/HttpResources.java) and [HttpResources master branch](https://github.com/reactor/reactor-netty/blob/45f8ad3c914abe95394c54b6a670f5fa7abad29c/src/main/java/reactor/netty/http/HttpResources.java). So both cases should be handled.